frontend/control-center/src/pages/Connectors/Providers/Facebook/Messenger/FacebookConnect.tsx

```diff
@@ -28,7 +28,7 @@ const FacebookConnect = (props: ConnectedProps<typeof connector>) => {
   const [token, setToken] = useState(channel?.metadata?.pageToken || '');
   const [name, setName] = useState(channel?.metadata?.name || '');
   const [image, setImage] = useState(channel?.metadata?.imageUrl || '');
-  const [buttonTitle, setButtonTitle] = useState(t('connectPage'));
+  const [buttonTitle, setButtonTitle] = useState(t<string>('connectPage'));
```

Comment:
So these type annotations are only necessary when we do
```typescript
import {t} from 'i18next';
```

rather than
```typescript
import {useTranslation} from 'i18next';

///...
const {t} = useTranslation();
```

Not sure why and how the first way works, but the second one looks more correct according to the docs so let's use that one